packages/date-time-picker/src/vaadin-date-time-picker.d.ts

```diff
@@ -87,14 +87,13 @@ export interface DateTimePickerEventMap extends DateTimePickerCustomEventMap, HT
  *
  * ### Internal components
  *
- * In addition to `<vaadin-date-time-picker>` itself, the following internal
- * components are themable:
+ * The following components are created by `<vaadin-date-time-picker>` and placed in light DOM:
  *
- * - `<vaadin-date-time-picker-date-picker>` - has the same API as [`<vaadin-date-picker>`](#/elements/vaadin-date-picker).
```

Comment:
Use of `vaadin-date-time-picker-date-picker` and `vaadin-date-time-picker-time-picker` still exist in `basic.test.js` and `outline.js` (field-highlighter)

Reply:
Good catch, fixed.